src/SprykerMiddleware/Zed/Process/Business/Writer/FileWriter.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace SprykerMiddleware\Zed\Process\Business\Writer;
+
+use Generated\Shared\Transfer\WriterConfigTransfer;
+
+class FileWriter implements WriterInterface
+{
+    /**
+     * @var string
+     */
+    protected $config;
+
+    /**
+     * FileWriter constructor.
+     *
+     * @param \Generated\Shared\Transfer\WriterConfigTransfer $writerConfigTransfer
+     */
+    public function __construct(WriterConfigTransfer $writerConfigTransfer)
+    {
+        $writerConfigTransfer->requireDestination();
+        $this->config = $writerConfigTransfer;
+    }
+
+    /**
+     * @param mixed $payload
+     *
+     * @return array
+     */
+    public function write($payload)
+    {
+        file_put_contents($this->config->getDestination(), $payload);
```

Comment:
Should be replaced with bridge to spryker/file-system?

Reply:
File writer should support multiple writes into existing file.

Reply:
Yes please.